src/styles/blocks/header.scss

```diff
@@ -0,0 +1,37 @@
+.header {
+  box-sizing: border-box;
+  background-image: url(../images/bo-mobile.png);
+  background-position: center;
+  background-size: cover;
+
+  &__content {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__bottom {
+    padding: 0 0 40px;
+
+    @include onTablet {
+      padding: 0 0 64px;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 36px;
+    line-height: 100%;
```

Comment:
Consider using a more appropriate line-height value, such as 1.2 or 1.4, instead of 100%.

Reply:
Sorry, but I think, line-height 100% is better for this title. Line-height changes with font-size for different media.